<ui version="4.0" >
 <class>SiteconBuildDialog</class>
 <widget class="QDialog" name="SiteconBuildDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>489</width>
    <height>243</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>sitecon_build_caption</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" >
   <item>
    <layout class="QGridLayout" name="gridLayout_2" >
     <item row="0" column="0" >
      <widget class="QLabel" name="inputLabel" >
       <property name="text" >
        <string>input_ali</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1" >
      <widget class="QLineEdit" name="inputEdit" />
     </item>
     <item row="0" column="2" >
      <widget class="QToolButton" name="inputButton" >
       <property name="text" >
        <string>...</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" >
      <widget class="QLabel" name="outputLabel" >
       <property name="text" >
        <string>out_profile</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1" >
      <widget class="QLineEdit" name="outputEdit" />
     </item>
     <item row="1" column="2" >
      <widget class="QToolButton" name="outputButton" >
       <property name="text" >
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="optionsBox" >
     <property name="title" >
      <string>options</string>
     </property>
     <layout class="QGridLayout" name="gridLayout" >
      <item row="0" column="0" >
       <widget class="QLabel" name="windowSizeLabel" >
        <property name="toolTip" >
         <string>win_tip</string>
        </property>
        <property name="text" >
         <string>window_size</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" >
       <spacer name="horizontalSpacer_2" >
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0" >
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="2" >
       <widget class="QSpinBox" name="windowSizeSpin" >
        <property name="minimumSize" >
         <size>
          <width>70</width>
          <height>0</height>
         </size>
        </property>
        <property name="toolTip" >
         <string>win_tip</string>
        </property>
        <property name="alignment" >
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="suffix" >
         <string>bp</string>
        </property>
        <property name="minimum" >
         <number>1</number>
        </property>
        <property name="maximum" >
         <number>1000</number>
        </property>
        <property name="value" >
         <number>40</number>
        </property>
       </widget>
      </item>
      <item row="1" column="0" >
       <widget class="QLabel" name="windowSizeLabel_2" >
        <property name="toolTip" >
         <string>seed_tip</string>
        </property>
        <property name="text" >
         <string>calibration_random_seed</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" >
       <spacer name="horizontalSpacer_3" >
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0" >
         <size>
          <width>207</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="2" >
       <widget class="QSpinBox" name="seedSpin" >
        <property name="minimumSize" >
         <size>
          <width>64</width>
          <height>0</height>
         </size>
        </property>
        <property name="toolTip" >
         <string>seed_tip</string>
        </property>
        <property name="alignment" >
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="suffix" >
         <string/>
        </property>
        <property name="minimum" >
         <number>0</number>
        </property>
        <property name="maximum" >
         <number>2000000000</number>
        </property>
        <property name="value" >
         <number>0</number>
        </property>
       </widget>
      </item>
      <item row="2" column="0" >
       <widget class="QLabel" name="windowSizeLabel_3" >
        <property name="toolTip" >
         <string>seq_len_tip</string>
        </property>
        <property name="text" >
         <string>calibration_sequence_len</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1" >
       <spacer name="horizontalSpacer_4" >
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0" >
         <size>
          <width>244</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="2" column="2" >
       <widget class="QComboBox" name="calibrationSeqLenBox" >
        <property name="toolTip" >
         <string>seq_len_tip</string>
        </property>
        <property name="layoutDirection" >
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="styleSheet" >
         <string notr="true" />
        </property>
        <property name="currentIndex" >
         <number>2</number>
        </property>
        <item>
         <property name="text" >
          <string>       100 Kb</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>       500 Kb</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>         1 Mb</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>         5 Mb</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="3" column="0" >
       <widget class="QLabel" name="label" >
        <property name="toolTip" >
         <string>algo_tip</string>
        </property>
        <property name="text" >
         <string>weight_alg</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1" >
       <spacer name="horizontalSpacer_5" >
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0" >
         <size>
          <width>234</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="3" column="2" >
       <widget class="QComboBox" name="weightAlgCombo" >
        <property name="toolTip" >
         <string>algo_tip</string>
        </property>
        <property name="layoutDirection" >
         <enum>Qt::LeftToRight</enum>
        </property>
        <item>
         <property name="text" >
          <string>       None</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>  Algorithm N2</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer" >
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>20</width>
       <height>10</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" >
     <item>
      <widget class="QLabel" name="statusLabel" >
       <property name="text" >
        <string>default_status</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_1" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="okButton" >
       <property name="text" >
        <string>start_button</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton" >
       <property name="text" >
        <string>cancel_butotn</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>SiteconBuildDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>395</x>
     <y>142</y>
    </hint>
    <hint type="destinationlabel" >
     <x>222</x>
     <y>82</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
